### PR TITLE
Minor changelog tweaks

### PR DESCRIPTION
**Pull request by eloquence**
Remove stray parentheses; expand "FPF"; use Packagecloud's own capitalization

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 since 0.4.1, and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Dangerzone 0.4.1]
+## Dangerzone 0.4.1
```
Comment (apyrgio) [inline CHANGELOG.md:8]: Good catch about the stray brackets. What I _should_ have done instead is the following, which is in line with the ["Keep a Changelog"](https://keepachangelog.com/en/1.1.0/) format.

```
## [Unreleased]

## [Dangerzone 0.4.1]

[... rest of file ...]

[Unreleased]: https://github.com/freedomofpress/dangerzone/compare/v0.4.1...HEAD
[Dangerzone 0.4.1]: https://github.com/freedomofpress/dangerzone/compare/v0.4.0...v0.4.1
```

Are you ok with adding this to the commit instead?

**Review by apyrgio — COMMENTED** [review]
Just one comment on the changelog links. Other than that, we can merge it.

File: CHANGELOG.md (hunk 1 above)
Comment (eloquence) [reply CHANGELOG.md]: I think it'd be good to standardize on that format throughout in later follow-up -- I've for now amended it to just add and bracket the "Unreleased"

Comment (apyrgio) [reply CHANGELOG.md]: We discussed about and we want to make as minimal changes as possible. So we will just add the "Unreleased" block, without the extra links.

**Pull request merged**